rs/replicated_state/tests/replicated_state.rs

```diff
@@ -998,223 +998,289 @@ fn compatibility_for_input_source() {
     );
 }
 
-proptest! {
-    #[test]
-    fn peek_and_next_consistent(
-        (mut replicated_state, _, total_requests) in arb_replicated_state_with_output_queues(SUBNET_ID, 10, 10, Some(5))
-    ) {
-        let mut output_iter = replicated_state.output_into_iter();
+#[test_strategy::proptest]
+fn peek_and_next_consistent(
+    #[strategy(arb_replicated_state_with_output_queues(SUBNET_ID, 10, 10, Some(5)))] test_input: (
+        ReplicatedState,
+        VecDeque<VecDeque<RequestOrResponse>>,
+        usize,
+    ),
+) {
+    let (mut replicated_state, _, total_requests) = test_input;
```

Comment:
Personal preference, feel free to ignore: I would name the input something like `state_and_queues` instead of the bland `test_input`.

Same elsewhere.